<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Chess</class>
 <widget class="QMainWindow" name="Chess">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>890</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chess</string>
  </property>
  <widget class="QWidget" name="_gameArea">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="20,4">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <item>
     <widget class="QGroupBox" name="_boardArea">
      <property name="title">
       <string>Board</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QWidget" name="_theGameBoard" native="true">
         <zorder>_capturedPiecesArea</zorder>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="_capturedPiecesArea">
      <property name="title">
       <string>Captured Pieces</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>1</number>
       </property>
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
       <item>
        <widget class="QGroupBox" name="_whitePiecesArea">
         <property name="title">
          <string>White</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QScrollArea" name="_whitePiecesScroller">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="_whitePiecesContainer">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>109</width>
               <height>269</height>
              </rect>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="_blackPiecesArea">
         <property name="title">
          <string>Black</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QScrollArea" name="_blackPiecesScroller">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="_blackPiecesContainer">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>109</width>
               <height>269</height>
              </rect>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>890</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_New_Game"/>
    <addaction name="separator"/>
    <addaction name="actionE_xit"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <action name="action_New_Game">
   <property name="text">
    <string>&amp;New Game</string>
   </property>
  </action>
  <action name="actionE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
